### DISCUSSION TITLE
Cool tools to help you on your journey

### DISCUSSION THREAD

**Discussion by drteresavasquez** [nss-evening-cohort-14/evening-client-side#5] [Discuss a Topic]
If anyone finds any cool tools, here is a good place to share them. Here also is a list of items that I have bookmarked:

- [Cohort 13 FE Issue Tickets](https://github.com/nss-evening-cohort-13/student-help/issues?q=is%3Aissue+is%3Aclosed) You can search these for some answers to common issues C13 had as well.
- [Curated Learner Resources](https://codetracker.netlify.app/resources)
- [Operator Lookup](https://joshwcomeau.com/operator-lookup/)
- [Dev Docs](https://devdocs.io/)
- [Markdown Cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
- [Emoji cheatsheet](https://www.webfx.com/tools/emoji-cheat-sheet/)
- [Console like a pro!](https://markodenic.com/use-console-log-like-a-pro/)

- [DOM EVENTS](https://domevents.dev)

React:
- [React Router](https://reactrouter.com/web/guides/quick-start)
- [React Router Cheat Sheet](https://drive.google.com/file/d/1ySy7igaShG_BtXHFL-oPxTeckyumiARj/view?usp=drivesdk)
- [React Selects](https://react-select.com/home)
- https://ondrabus.com/pdf/cheatsheet-react.pdf

**Comment by jrobinson0529** [comment]
### These are great, thanks!

I like using this for colors when making a site. Also added the pdf that Jim originally linked in Slack.

*  [Html Color Picker](https://htmlcolorcodes.com/color-picker/)
*  [VSC Hotkey PDF](https://code.visualstudio.com/shortcuts/keyboard-shortcuts-windows.pdf) -- **Windows only** Courtesy of [Jim Connor](https://github.com/jim-conner)
*  [Git Workflow PDF](https://nashvillesoftware.slack.com/files/UQW9WM9TR/F01JDG5HYRK/git-workflow.pdf) -- Courtesy of [Holly Parsons](https://github.com/hmparsons-sde)

**Reply by jimguistical** [reply]
Thank you for posting that + the color picker!

**Comment by lindseysatterfield** [comment]
Mac shortcuts for VS Code
![VS Code Mac shortcuts](https://user-images.githubusercontent.com/23327097/104106259-18c54000-527a-11eb-9f65-cadb14a81c06.png)

**Comment by drteresavasquez** [comment]
Have added a few things here! Make sure to add things you come across!

**Comment by drteresavasquez** [comment]
![image](https://user-images.githubusercontent.com/29741570/117528748-68521900-af99-11eb-9c00-de1565b13661.jpeg)
